src/odemis/gui/model/__init__.py

```diff
@@ -596,7 +603,56 @@ def __init__(self, main):
         self.autofocus_active = BooleanVA(False)
 
 
-class LocalizationGUIData(MicroscopyGUIData):
+class FeaturesListVA(model.ListVA):
+    """
+    A singleton class to share the feature list on all cryo tabs
+    """
+    __instance = None
+
+    def __init__(self, value=None, *args, **kwargs):
+        # Copy already connected listeners as it will be emptied on VA init
+        listeners = set()
+        if hasattr(self, "_listeners"):
+            listeners = self._listeners
+        model.ListVA.__init__(self, value, *args, **kwargs)
+        self._listeners = listeners
+
+    def __new__(cls):
+        if FeaturesListVA.__instance is None:
+            FeaturesListVA.__instance = object.__new__(cls)
+        return FeaturesListVA.__instance
+
+class CryoGUIData(MicroscopyGUIData):
+    """
+    Represents an interface for handling cryo microscopes.
+    It's meant to be subclassed from the cryo tabs, to share data between all the tabs (like features)
+    """
+    def __init__(self, main):
+        if "cryo" not in main.role:
+            raise ValueError(
+                "Expected a cryo microscope role but found it to be %s." % main.role)
+        MicroscopyGUIData.__init__(self, main)
+        # List VA contains all the CryoFeatures (for the current project).
+        self.features = FeaturesListVA()
+        # VA for the currently selected feature
+        # TODO: Make it also a singleton
+        self.currentFeature = model.VigilantAttribute(None)
```

Comment:
The code to make it a singleton is quite tricky (especially with the listeners), and it doesn't even support the case of currentFeature.
Let's just move these 2 VAs to MainGUIData. I see there are already some shared VA like `fineAlignDwellTime`, and it's all fine. If you want, you could put it in a `if "cryo" not in main.role:` block, so that they only appear on cryo.